aiida/orm/implementation/backends.py

```diff
@@ -149,3 +204,30 @@ def delete_nodes_and_connections(self, pks_to_delete: Sequence[int]):
     @abc.abstractmethod
     def get_repository(self) -> 'AbstractRepositoryBackend':
         """Return the object repository configured for this backend."""
+
+    # TODO need to decide if `set_global` and `get_global` are ok for the `Backend` abstract.
+    # These methods allow access to the `db_dbsetting` table, which is not currently possible via the AiiDA ORM
```

Comment:
I was wondering this as well. But I guess that this kind of makes sense. If there ever were another backend that doesn't necessarily use a RDM and so that might not have the settings table, it still needs a way to store global info, such as the schema version it is at. Might make sense to keep it here for now.

Reply:
Are you happy with the names `get_global`, `set_global`?
Otherwise I will remove this TODO then.
Copying the comment from it here for prosperity:

>  These methods allow access to the `db_dbsetting` table, which is not currently possible via the AiiDA ORM
    they are currently only used in `get_process_state_change_timestamp` and `set_process_state_change_timestamp`

Reply:
Since it is again mostly internal we can always change so don't want this to keep things up, but maybe `get/set_storage_setting` would be a bit clearer? Already preparing for the `backend` -> `storage` rename?

Reply:
I wouldn't include `setting` in the name because, despite the table being named as such, it's actually just for storing any "global" value for the profile, e.g. `process|state_change`

Reply:
Not suggesting `settings` because that is what the table is currently called, but simply because just `global` seems a bit vague. A global what? If you can find another word that works well, that's fine as well.